### DISCUSSION TITLE
Sidebar configuration error

### DISCUSSION THREAD

**Discussion by ilg-ul** [tgreyuk/typedoc-plugin-markdown#706] [Help]
At the end of https://typedoc-plugin-markdown.org/plugins/docusaurus/options there is a short code section demonstrating how to add the `sidebar` configuration to `typedoc.json`.

I tried this in a Docusaurus setup and got:

```
[INFO] Starting the development server...
[info] Loaded plugin typedoc-plugin-markdown
[error] Unknown option 'sidebar' You may have meant:
        sidebarLinks
        sort
[error] Documentation could not be generated due to the errors above
```

Without the `sidebar` definition, Docusaurus starts properly:

```
[INFO] Starting the development server...
[info] Loaded plugin typedoc-plugin-markdown
[info] Documentation generated at ./docs/api
[SUCCESS] Docusaurus website is running at: http://localhost:3000/logger-ts/
```

**Comment by tgreyuk** [comment]
Hi @ilg-ul. Something not right here. I don't suppose your project is on a public repo?

**Reply by ilg-ul** [reply]
It is public:

- https://github.com/xpack/logger-ts/

I'm migrating the web from the pages in `docs`, generate with TypeDoc, to those in `website`, a Docusaurus configuration.

The migration is _work in progress_, it will hopefully be ready in a few more days.
